### ISSUE TITLE
CMake build failure on non-x86-64 hosts

### ISSUE THREAD

**Issue by Nebuleon** [gonetz/GLideN64#811]
GLideNHQ bundles its own [statically-linked versions of libraries such as libpng and libz](https://github.com/gonetz/GLideN64/tree/28d4993968ec1aafa74a198c435b10839023338a/src/GLideNHQ/lib). However, these are only usable on x86-64, with linker errors occurring on any other architecture, for both native compilation and cross-compilation with CMake.

It would be nice if the default were switched to try to use the system's libpng or libz, statically or dynamically linked, with `FIND_PACKAGE`, as if `BCMHOST` were always defined in these two blocks:

[commit 28d4993's `GLideNHQ/CMakeLists.txt`, lines 93-108](https://github.com/gonetz/GLideN64/blob/28d4993968ec1aafa74a198c435b10839023338a/src/GLideNHQ/CMakeLists.txt#L93-L108), for debug

[commit 28d4993's `GLideNHQ/CMakeLists.txt`, lines 119-141](https://github.com/gonetz/GLideN64/blob/28d4993968ec1aafa74a198c435b10839023338a/src/GLideNHQ/CMakeLists.txt#L119-L141), for release

Is the bundled version actually required for some compilation targets?

**Comment by gonetz** [comment]
libpng is very outdated on Ubuntu 14. GLideNHQ needs more fresh version, thus I built it and libz from sources.
I think, it is possible to include alternative repository with more fresh version of these libs and use system libs instead of my builds. I did not investigate that question. I use Windows for development.
Linux users usually smart enough to solve problems with builds ;) I just made some initial solution, which works for me. If you know a simple recipe how to get fresh version of libpng on Ubuntu (at least libpng 1.44), it will solve the problem.
